go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/sustainable-computing-io/kepler
 
 go 1.18
 
+replace github.com/iovisor/gobpf => github.com/rootfs/gobpf v0.2.1-0.20230501231255-cb9cf9f81504
```

Comment:
it relays on specific version?

Reply:
it points to the `batch` branch

Reply:
do you know if there are any plan for gobpf merge `batch` branch with a new release?

Reply:
let's discuss it in the planning meeting.